public/keycloak/keycloak.ts

```diff
@@ -139,8 +143,20 @@ class KeyCloakService {
     if (!isForceRefresh && (!this.kc?.tokenParsed?.exp || !this.kc.timeSkew)) {
       return
     }
+
+    let tokenholder = null;
     // if isForceRefresh is true, send -1 in updateToken to force update the token
-    const tokenExpiresIn = (isForceRefresh) ? -1 : this.kc.tokenParsed.exp - Math.ceil(new Date().getTime() / 1000) + this.kc.timeSkew + 100
+    if (isForceRefresh){
+      tokenholder = -1;
+    }
+    else{
```

Comment:
Lint should have cleaned this up, too (`else {`)...

Also unneeded semi-colon on line 158 below.